### PR TITLE
feat: add skeleton loading & fix layout for login card

### PR DESCRIPTION
**Pull request by RaymondWHZ**
I forgot to add skeleton loading and adjust the login card layout for the footer in previous PRs. Now it looks like this:

Screenshot 2023-06-30 at 9 55 45 AM

### CODE REVIEW

File: src/components/section/LogInCard.tsx [hunk 1]
```diff
@@ -21,50 +21,58 @@ export const LogInCard: React.FC = () => {
...
+                        // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+                        event.timeZone
+                      )
+                    : "Loading..."}
+                </div>
+                <MdOutlineAccessTime className="ml-1" />
+                <div>
+                  {event
+                    ? // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+                      formatTimespan(event.begins, event.ends, event.timeZone)
```
Comment (louisunlimited) [inline src/components/section/LogInCard.tsx:45]: Why are we ignoring these? btw when I remove these I don't get eslint errors🤔

File: src/components/section/LogInCard.tsx [hunk 2]
```diff
@@ -21,50 +21,58 @@ export const LogInCard: React.FC = () => {
...
               </div>
             </div>
-            <div className="text-3xl font-semibold text-black">
-              {event?.name ?? "Loading..."}
+          ) : (
+            <div className="flex flex-1 flex-col gap-2 border-b-[1px] border-gray-300 p-5">
+              <div className="skeleton h-5 max-w-[400px]" />
+              <div className="skeleton h-9 max-w-[600px]" />
+              <div className="skeleton h-5 max-w-[300px]" />
             </div>
```
Comment (louisunlimited) [inline src/components/section/LogInCard.tsx:74]: A left to right running highlight on the gray skeleton would be much better imo, but this looks fine as well!

File: src/components/section/LogInCard.tsx (hunk 1 above)
Comment (RaymondWHZ) [reply src/components/section/LogInCard.tsx:45]: We used to need them, but not now. I have removed them.

File: src/components/section/LogInCard.tsx (hunk 2 above)
Comment (RaymondWHZ) [reply src/components/section/LogInCard.tsx:74]: Making such animation requires more than just a Tailwind CSS class as far as I know. We can leave this to a later update.

**Review by louisunlimited — APPROVED** [review]
Great work!!

**Pull request merged**